src/ripple/ledger/impl/CachedView.cpp

```diff
@@ -33,25 +33,38 @@ CachedViewImpl::exists(Keylet const& k) const
 std::shared_ptr<SLE const>
 CachedViewImpl::read(Keylet const& k) const
 {
-    {
+    static CountedObjects::Counter hits{"CachedView::cache_hit"};
+    static CountedObjects::Counter hitsexpired{"CachedView::cache_hit_expired"};
```

Comment:
this name irks me, unsure why. Can we give it capitalised "E", like so: `hitsExpired` ?

Reply:
Sure. We can call it whatever makes sense. I chose the name based on the other names that appear at the top of a `get_counts` call:
```
      "AL_hit_rate" : 38.9652214050293,                                                        
      "AL_size" : 50,                          
      "SLE_hit_rate" : 0.9263432896252034,
      "dbKBLedger" : 6469,         
      "dbKBTotal" : 142664,                                                                    
      "dbKBTransaction" : 134785,         
      "fullbelow_size" : 1066268,     
      "historical_perminute" : 21,     
      "ledger_hit_rate" : 4.897829055786133,                                                   
      "node_read_bytes" : "3305869523",                                                        
      "node_reads_duration_us" : "72651617585", 
      "node_reads_hit" : "198704709",
      "node_reads_total" : "267571236",                                                        
      "node_writes" : "56873374",                                                              
      "node_written_bytes" : "28242167146",
[...]
```
But I don't have a problem using `hitsExpired`, etc.